bookkeeper-server/src/main/java/org/apache/bookkeeper/conf/ClientConfiguration.java

```diff
@@ -1618,6 +1619,26 @@ public ClientConfiguration setDisableEnsembleChangeFeatureName(String disableEns
         return this;
     }
 
+    /**
+     * Get the MaxNum for ensemble change.
+     *
+     * @return value of MaxNumEnsembleChanges, default 100.
+     */
+    public int getMaxNumEnsembleChanges() {
+        return getInt(MAX_NUM_ENSEMBLE_CHANGES, 100);
```

Comment:
make default as Integer.MAX_VALUE, so by default this feature is *OFF*. if users want to use this feature, he can turn on by setting it to a lower value.